lib/tasks/scripts/message_patients.rb

```diff
@@ -39,22 +39,18 @@ def call
   def send_messages
     contactable_patients.each do |patient|
       phone_number = phone_number_for(patient)
-
       next unless phone_number
-      notification_service = TwilioApiService.new
-      context = {
-        calling_class: self.class.name,
-        patient_id: patient.id,
-        communication_type: channel
-      }
 
-      begin
-        response = if whatsapp?
-          notification_service.send_whatsapp(recipient_number: phone_number, message: message, context: context)
+      messaging_channel =
+        if whatsapp?
+          Messaging::Twilio::Whatsapp
         elsif sms?
-          notification_service.send_sms(recipient_number: phone_number, message: message, context: context)
+          Messaging::Twilio::ReminderSms
         end
-      rescue TwilioApiService::Error
+
+      begin
+        response = messaging_channel.new.send_message(recipient_number: phone_number, message: message)
```

Comment:
Can we have the channels have a class method that delegates to an instance to make this interface a bit easier?
`messaging_channel.send_message(...)`

Reply:
Addressed in [2aa5adc](https://github.com/simpledotorg/simple-server/pull/3589/commits/2aa5adc2d864bf02037d38a6768d5680beec1505)